bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="freeboxos" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Freebox OS Binding</name>
+	<description>The Freebox OS binding integrates Free equipments in your home automation.</description>
+	<connection>local</connection>
+	<countries>fr,it</countries>
+
+	<config-description>
+		<parameter name="timeout" type="integer" required="false" min="1" max="30" unit="s">
```

Comment:
Is it reasonable to specify a max value?

Reply:
You're correct